### ISSUE TITLE
Components in custom modules paths are missing styles with 2.0

### ISSUE THREAD

**Issue by ippeiukai** [ionic-team/ionic-app-scripts#1090]
#### Short description of the problem:

We have ionic components in custom modules paths configured in webpack.config.js. The scss files in those components were included in main.css prior to ionic-app-scripts 2.0 update, but no longer.

#### What behavior are you expecting?

One can use Ionic style .scss files in components that are neither in `node_modules` nor in `SRC_DIR`.

This was possible in previous versions by adding a modules dir in webpack.config.js. However, as long as there is a way to configure such directories, I would be happy to switch over.

**Steps to reproduce:**

We have components in custom paths configured as follows in webpack.config.js:
```JS
  resolve: {
    extensions: ['.ts', '.js', '.json'],
    modules: [
      path.resolve('node_modules'),
      path.resolve('../../shared'),
    ],
  },
```

The `../../shared` directory above is next to the apps (`..`) directory in which the apps reside, and hosts the components shared with other apps in the same repo.

With `ionic serve`, per-component styles specified in files such as `../../shared/pages/login/login.scss` are expected to be included in `www/build/main.css`, but not after app-scripts 2.0 update. 

**Which @ionic/app-scripts version are you using?**

- 1.3.12 works
- 2.0.0 does not

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

The components in question are in neither `node_modules` nor `SRC_DIR`, but included in another modules dir specified to webpack.config.js. The changes made in 2.0 started to make an assumption that the components in `context.moduleFiles` (for which scss files are processed) are only in those two directories, breaking my use case:

https://github.com/ionic-team/ionic-app-scripts/commit/e14f81944c188186affe9f07f69a1b107f3fb38f#diff-a9f426388655ad21e8c5d39378c0d070L96
↓
https://github.com/ionic-team/ionic-app-scripts/commit/e14f81944c188186affe9f07f69a1b107f3fb38f#diff-384a98a03c3b07cdf1d7df84e03f8465R72

If specifying such directories only in `webpack.config.js` is no longer an option, I would love to add an additional configuration (e.g. ionic_additional_modules_dirs in package.json) to app-scripts, which then by default would be passed to both webpack and preprocess (and in turn to scss via `context.moduleFiles`).

**Comment by emiljohansson** [comment]
@ippeiukai Might not be related, but it could be worth a try to upgrade to 2.0.1. It resolved our sass issues we faced with 2.0.0

**Comment by ippeiukai** [comment]
2.0.2 LGTM 👍
